tools/fixup-wheels.sh

```diff
@@ -82,10 +82,12 @@ fixup_wheel_macosx () {
 
         # Change the name to be local
         install_name_tool -id "$lib" $lib
-        if otool -L $lib | grep -q /usr/local/lib/librdkafka.1.dylib ; then
+        # Extract existing(old) reference
+        old=$(otool -L $lib | grep -o .*librdkafka.1.dylib)
```

Comment:
You need to use single quotes around the regex to avoid shell expansion of `*`.

also, technically, the returned string will have a bunch of leading spaces that should be trimmed off.